docs/rules/indent.md

```diff
@@ -305,6 +305,37 @@ if(y) {
 }
 ```
 
+Examples of **correct** code for this rule with the options `2, { "outerIIFEBody":  "off" }`:
+
+```js
+/*eslint indent: ["error", 2, { "outerIIFEBody": "off" }]*/
+
+(function() {
+
+function foo(x) {
+  return x + 1;
+}
+
+})();
+
+(function() {
+
+  function foo(x) {
+    return x + 1;
+  }
+
+})();
+
+
+if (y) {
+console.log('foo');
```

Comment:
This seems strange to me. This is not an IIFE body, so the indentation should be required on this line... Shouldn't it? Or am I missing something?

Reply:
Huh, this looks like a typo. Thanks for catching this!